Hello Microverse Project

dielupovissa opened the pull request:
# Changes made so far:
- Added a readme file
- Added a gitgnore file
- Added an HTML/CSS file
- Added a text file: it was set up to be ignored by git

dielupovissa committed:
Update README.md

dielupovissa committed:
Update README.md

dielupovissa committed:
Added a html file

dielupovissa committed:
Added a file .css

dielupovissa committed:
Added test.md

dielupovissa committed:
Added files

dielupovissa committed:
Added Codes in Files html/Css

RolandM99 commented on README.md:
```diff
@@ -1,2 +1,31 @@
 # Hello-Microverse
 This is a hello microverse project for my course in Microverse.
+In this project, you will set up a "Hello world" repository. 
+No complex coding is required for this exercise. 
```
- [ ]  Your README looks great 👍🏻  but I think you can improve it by making it more professional. A professional README file should have a project title, project description, list of technology used in the project, and instructions to set the project up locally. please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

sja-thedude replied:
Good job implementing the changes asked by the previous reviewer. 👍 
Your readme is perfect!

RolandM99 commented on README.md:
```diff
@@ -1,2 +1,31 @@
+
+## Build With
+- HTML/CSS
+- Linter: github actions
+## General Requirements
```
Great Job by seting up  Github actions for your project. Unfortunally, all Linters checks have failed. It seems that you didn't set up them correctly.
![image](https://user-images.githubusercontent.com/71874425/158417465-c3149a0d-7898-40b6-9919-ec5b98769fba.png)
To fix these issues, Kindly follow correctly the steps listed in this [documentation](https://github.com/microverseinc/linters-config/tree/master/html-css)

sja-thedude replied:
All linters are passing! Great job! 👍

RolandM99 reviewed (changes requested):
Hi @dielupovissa  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)

### To highlight 💯 
- Follow Github flow ✔️ 

## Required Changes ♻️


- [ ] Make sure that your Pull Request has a descriptive title and tells us what changes are inside. Please, kindly fix this issue by editing your Pull Request. 
- [ ] You don't need to create for each requirement in this project a new branch, kindly add all requirements(HTML file, CSS file, test.md file,...) in this feature branch so that we can see all features you implemented for this milestone.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

dielupovissa committed:
Update README.md

# Changes made:
- Added a icone of microverse.
- Added an author field.
- Added a contributing and support field.

dielupovissa committed:
Update test.md

Added a sentence in the text.

dielupovissa committed:
Create .gitignore

dielupovissa committed:
Update .gitignore

dielupovissa committed:
changed the location of the files

dielupovissa committed:
changed the location of the files

dielupovissa committed:
Update index.html

Added a tag title and a tag lang in html tag.

dielupovissa committed:
Update style.css

Added indentation

dielupovissa committed:
Update style.css

Added a space before {.
Added a indentation of two spaces

dielupovissa committed:
Update of html/css files

dielupovissa committed:
Updated html/css files

dielupovissa committed:
Removed the test.md

sja-thedude reviewed (changes requested):
## Required Changes ♻️

Hi @dielupovissa,  This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required: 

- [ ] Kindly replace your Pull request title with `Hello Microverse Project` which will make it clear what you are working on.


 ## Highlights
- All linters are passing 👍 
- Professional Readme 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

dielupovissa commented:
I did the change of the title of pull reques as you mention @sja-thedude is there anything alse to change?

sja-thedude commented:
No @dielupovissa, everything is okay. You can submit it for review.

dielupovissa commented:
thanks

dielupovissa commented:
I cannot submit it!

dielupovissa commented:
@sja-thedude is there yet a review pending.

ShadyShawkat reviewed (approved):
## Hi @dielupovissa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Image](https://media.giphy.com/media/qlrBlSDevEdFeW5JwV/giphy.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

dielupovissa commented:
Thank you! What link I have to submit @ShadyShawkat ?

ShadyShawkat commented:
Hey @dielupovissa, you can submit the link to the repo after the approval for your project.
